scripts/dotnet-cli-build/TestTargets.cs

```diff
@@ -178,7 +165,7 @@ public static BuildTargetResult BuildTestAssetPackages(BuildTargetContext c)
                 {
                     "netstandard1.5",
                     "netstandard1.3",
-                    "netstandardapp1.5"
+                    "netcoreapp1.0"
```

Comment:
I still don't think I get it.  We build all these packages on all these frameworks just so we can ignore net451 build failures on non-windows?

Maybe it would be better if the tests that needed/used these projects would do the building during the test, instead of building them during this step in the build?